### PR TITLE
[FEAT]: Implement Shape Drawing Tools #33

### PR DESCRIPTION
**Pull request by Prithwi32**
# Issue Title: [FEAT]: Implement Shape Drawing Tools
*resolve issue:* #33 

## Type of change ☑️
-Drawing shapes like: Rectangle, Circle, Triangle with selecting colours
-with fill colour and without fill colour

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: ☑️
- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

 
## How Has This Been Tested? ⚙️
Tested using localhost from vscode editor with all related possibility

### CODE REVIEW

**Comment by Prithwi32** [comment]
@singodiyashubham87 Made PR ..go through it and let me know feedback

**Comment by Prithwi32** [comment]
> video link: [](https://drive.google.com/file/d/1IuAGwDYxHO3P5mMvNGvHZ4ixky7657wK/view?usp=sharing)
@singodiyashubham87 Made PR ..go through it and let me know feedback

**Review by 0xabdulkhalid — APPROVED** [review]
LGTM 👍🏻 

Good to go 🚀

**Comment by singodiyashubham87** [comment]
@Prithwi32 Kindly resolve the conflicts so that this PR can be merged.

**Comment by Prithwi32** [comment]
@singodiyashubham87 Hello..could you check out now

**Comment by singodiyashubham87** [comment]
@Prithwi32 I reviewed your PR, everything looks fine except that you've used custom css instead of tailwindCSS classes which is not what the tech-stack of project is, Kindly remove the custom css file "drawShapes" and implement that styling using TailwindCSS classes only.

**Comment by Prithwi32** [comment]
> Hi there @Prithwi32 👋🏻
> 
> I've seen that you removed the `drawShapes.css` file, but you haven't added the respective styles using TailwindCSS yet.
> 
> Once you added Tailwind classes, then your PR is ready to be merged!

hey actually that respective CSS is already in DrawShape.jsx file...the custom CSS file I was added before when working ..I forgot to remove ..

File: src/components/DrawingShapes.jsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
+import { useEffect, useState } from "react";
+import "../style/drawShapes.css";
```
Comment (0xabdulkhalid) [inline src/components/DrawingShapes.jsx:2]: Can you please remove this line ?

You can't import a file which not exists, Due to that Vercel is failing to deploying the application.

Screenshot 2024-05-18 at 22-45-53 Dashboard

**Review by 0xabdulkhalid — CHANGES_REQUESTED** [review]

File: src/components/DrawingShapes.jsx (hunk 1 above)
Comment (Prithwi32) [reply src/components/DrawingShapes.jsx]: thank you :) 
done, if u can remove circle.png also

**Review by 0xabdulkhalid — APPROVED** [review]
Ready to be Merged @singodiyashubham87 🚀

**Comment by singodiyashubham87** [comment]
@Prithwi32 Well Done!
Keep up the good work :)

**Pull request merged**